src/olympia/devhub/views.py

```diff
@@ -1166,11 +1167,23 @@ def version_edit(request, addon_id, addon, version_id):
                         'max' in form.changed_data):
                     _log_max_version_change(addon, version, form.instance)
 
-        fields = ('source', 'approvalnotes')
-        has_changed = [field in version_form.changed_data for field in fields]
-        if version.has_info_request and any(has_changed):
-            version.update(has_info_request=False)
-            version.save()
+        if 'approvalnotes' in version_form.changed_data:
+            if version.has_info_request:
+                version.update(has_info_request=False)
+                log_and_notify(amo.LOG.APPROVALNOTES_CHANGED,
```

Comment:
You need to put the use of `log_and_notify` behind the `activity-email` waffle switch for now.  I'd just create the log() if the waffle is off.